File: joseffe-barroso/Java/src/entrada_processamento_saida/aula04/While/Exercise36.java
package While;

import java.util.Scanner;

public class Exercise36 {

	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);

		int value, interval, index, result;

		System.out.print("Informe um valor qualquer: ");
		value = scanner.nextInt();

		while (value <= 0) {
			System.out.print("Informe um valor positivo: ");
			value = scanner.nextInt();
		}

		System.out.print("Informe o intervalo: ");
		interval = scanner.nextInt();

		while (value > interval) {
			System.out.print("O intervalo dever ser maior que o valor informado: ");
			interval = scanner.nextInt();
		}

		index = interval;

		while (index > 0) {
			result = value * index;
			System.out.printf("\n%d X %d = %d", value, index, result);
			index = index - 1;
		}
		
		scanner.close();

	}

}

File: joseffe-barroso/Python/aula04/for/exercise36.py
from tracemalloc import start


value = int(input("Informe um valor qualquer: "))

while (value <= 0): 
    value = int(input("Informe um valor positivo: "))

startInterval = int(input("Informe o ínicio do intervalo: "))
endInterval = int(input("Informe o final do intervalo: "))

while (endInterval <= startInterval): 
    endInterval = int(input("O valor final do intervalo deve ser maior que o valor inicial: "))

for index in range (startInterval, endInterval + 1, 1):
    result = value * index
    print(f"{value} X {index} = {result}")
    index = index + 1
